specifications/secure-launch-specification.rst

```diff
@@ -478,6 +478,10 @@ environment, that attribute will be published as an entity type. A generic
         char evt_info[TPM_EVENT_INFO_LENGTH];
     };
 
+In the current version (one) of the specification, `TPM_EVENT_INFO_LENGTH` is
+defined as 32 bytes. All unused bytes **MUST** be set to `\0`, but the string
+**MAY** not be terminated with `\0` if it fills the whole `evt_info`.
+
```

Comment:
This block should be moved to above the fields definitions and made into a `.. note::` block